adding a new post for diabetic eye screening (#478)
NHSDigital/prevention-services-design-history: app/diabetic-eye-screening/2025/13/From-Co-design-to-Delivery-A-Phased-Approach-to-Improving-Diabetic-Eye-Screening/index.md

Reviewer: Please rename this folder to be all lowercase. You could consider shortening it too - `from-co-desgin-to-delivery` or similar.

Reviewer: This whole post needs to be in a folder '2026' and then a folder '04' - not 2025/13

---
title: From Co-design to Delivery - A Phased Approach to Improving Diabetic Eye Screening

> description: A phased, clinically assured approach to delivering improvements in Diabetic Eye Screening cohorting

Reviewer: ```suggestion
title: From co-design to delivery - a phased approach to improving diabetic eye screening
description: A phased, clinically assured approach to delivering improvements in diabetic eye screening cohorting
```

NHS style guide is to use sentence case, not title case.

> date: 2026-04-23

Reviewer: ```suggestion
date: 2026-04-24
```

author: Harleen Kaur Dhillon

> tags: CASCADE

Reviewer: I think our standard is tags as all lowercase

>   

Reviewer: Would probably be good to link the image here as an opengraph image. I can't see what the image is though, so only link if relevant. You may need to update the src url depending on where you move this folder.

```suggestion
opengraphImage:
  src: /diabetic-eye-screening/2026/04/from-co-design-to-delivery/pilot-approach.png
  alt: high level pilot approach starting from phase one to phase five
```

---

> To ensure a safe, effective, and practical Diabetic Eye Screening (DES) cohorting process, DES and CaaS are working with Diabetic Eye Screening Providers (DESPs), system suppliers, and stakeholders through a phased approach. This enables early and ongoing collaboration to validate assumptions, test operational impacts, and shape design and implementation.

Reviewer: ```suggestion
To ensure a safe, effective, and practical diabetic eye screening (DES) cohorting process, DES and CaaS are working with diabetic eye screening providers (DESPs), system suppliers, and stakeholders through a phased approach. This enables early and ongoing collaboration to validate assumptions, test operational impacts, and shape design and implementation.
```

If you were referring to the service I might capitalise diabetic - but here it seems to just be words about the screening - so should all be sentence case.

> To ensure a safe, effective, and practical Diabetic Eye Screening (DES) cohorting process, DES and CaaS are working with Diabetic Eye Screening Providers (DESPs), system suppliers, and stakeholders through a phased approach. This enables early and ongoing collaboration to validate assumptions, test operational impacts, and shape design and implementation.

Reviewer: Who are CaaS? Can the acronym be spelt out?

A core principle of the programme is simple: avoid large-scale change without evidence. Instead, incremental improvements are introduced through a controlled pilot model. This ensures system behaviour, workflows, and clinical safety are fully understood before wider rollout. Clinical assurance is embedded from the outset, with cohorts validated in the first phase rather than later in delivery

> The CASCADE (CaaS Cohorting and Diabetic Eye) Operational Working Group has been established to bring together operational, clinical, QA, and pilot site stakeholders. Regular sessions provide a forum to review pilot progress, assess impacts, and support informed decision-making.

Reviewer: ```suggestion
The CASCADE (CaaS Cohorting and Diabetic Eye) operational working group has been established to bring together operational, clinical, quality assurance, and pilot site stakeholders. Regular sessions provide a forum to review pilot progress, assess impacts, and support informed decision making.
```

> A collaborative workshop held in March in Leeds supported Phase 2 planning, ensuring alignment on system requirements, integration dependencies, and delivery sequencing.

Reviewer: ```suggestion
A collaborative workshop held in March in Leeds supported phase 2 planning, ensuring alignment on system requirements, integration dependencies, and delivery sequencing.
```

> A collaborative workshop held in March in Leeds supported Phase 2 planning, ensuring alignment on system requirements, integration dependencies, and delivery sequencing.

Reviewer: Feels a bit weird to talk about 'phase 2' before mentioning what the phases are.

> The approach builds on the existing GP2DRS process, improving data quality, timeliness, and integration with national services such as the Personal Demographic Service (PDS).

Reviewer: What is GP2DRS?

Delivery Plan Overview
Phase 1. cohort definition and validation
Phase 2. technical integration
Phase 3. controlled pilot testing
Phase 4. rollout planning

> Phase 5. full implementation and transition

Reviewer: I wonder if this could be reworked to use an ordered lists.
```suggestion
Delivery plan phases:

1. Cohort definition and validation
2. Technical integration
3. Controlled pilot testing
4. Rollout planning
5. Full implementation and transition
```

> ## Key Phases of the Delivery Approach

Reviewer: ```suggestion
## Key phases of the delivery approach
```

Sentence case

> ![high level pilot approach starting from phase one to phase five ](pilot-approach.png)

Reviewer: Could the alt text say a bit more about what the image shows?

> ![high level pilot approach starting from phase one to phase five ](pilot-approach.png)

Reviewer: This image isn't included in the PR so is broken.

### Phase 1. cohort definition and validation
This phase focuses on defining and validating cohorts that accurately reflect real-world eligibility in collaboration with DESPs. Outputs are reviewed to ensure alignment with clinical and operational requirements, including resolution of discrepancies between GP and PDS data. Daily cohort refreshes provide timely updates on patient eligibility. The phase concludes when the cohort is clinically assured and accepted by pilot DESPs.

### Phase 2. technical integration
This phase establishes the technical foundation for the cohorting model. Collaborative workshops define integration requirements across NHS England systems, suppliers, and services such as PDS. While DESPs do not directly interact with cohort data at this stage, their input ensures operational needs are met. The phase concludes when data exchange is fully operational across systems.

### Phase 3. controlled pilot testing with DESPs

> A controlled pilot involving four DESPs evaluates the cohorting process using production-level data. The pilot evaluates system behaviour under dynamic conditions, impacts of eligibility changes, and workflow adaptations. The CASCADE Working Group supports ongoing review and feedback. The phase concludes once an end-to-end viable and accepted solution is achieved.

Reviewer: ```suggestion
A controlled pilot involving four DESPs evaluates the cohorting process using production-level data. The pilot evaluates system behaviour under dynamic conditions, impacts of eligibility changes, and workflow adaptations. The CASCADE working group supports ongoing review and feedback. The phase concludes once an end-to-end viable and accepted solution is achieved.
```

### Phase 4. rollout planning with stakeholders

> Following successful pilot completion, NHS England works with DESPs and suppliers to plan national rollout. Local variations in processes and system configurations are incorporated. Insights from the CASCADE Working Group ensure a clear and actionable rollout strategy.

Reviewer: ```suggestion
Following successful pilot completion, NHS England works with DESPs and suppliers to plan national rollout. Local variations in processes and system configurations are incorporated. Insights from the CASCADE working group ensure a clear and actionable rollout strategy.
```

### Phase 5. full implementation and transition
The final phase scales the process across all DESPs. Automated cohort updates replace manual processes and legacy systems. The approach has been validated through pilot sites, supporting confident national adoption. Full rollout is targeted for March 2027.

> ## Collaborative governance and working pproach

Reviewer: ```suggestion
## Collaborative governance and working approach
```

> The CASCADE Operational Working Group meets regularly, bringing together operational, clinical, QA, and pilot site representatives. It provides a structured forum for reviewing progress, assessing risks, and supporting decisions throughout pilot and rollout phases.

Reviewer: ```suggestion
The CASCADE operational working group meets regularly, bringing together operational, clinical, QA, and pilot site representatives. It provides a structured forum for reviewing progress, assessing risks, and supporting decisions throughout pilot and rollout phases.
```

> Clinical governance is being strengthened through the involvement of the CaaS Chief Clinical Safety Officer (CSO), ensuring alignment with existing CaaS clinical safety oversight. Supplier clinical safety leads, including those from NEC and InHealth Intelligence, are also being engaged, with initial alignment discussions underway to ensure a shared understanding of the cohorting approach, associated risks, and safety responsibilities.

Reviewer: ```suggestion
Clinical governance is being strengthened through the involvement of the CaaS chief clinical safety officer (CSO), ensuring alignment with existing CaaS clinical safety oversight. Supplier clinical safety leads, including those from NEC and InHealth Intelligence, are also being engaged, with initial alignment discussions underway to ensure a shared understanding of the cohorting approach, associated risks, and safety responsibilities.
```

> Clinical governance is being strengthened through the involvement of the CaaS Chief Clinical Safety Officer (CSO), ensuring alignment with existing CaaS clinical safety oversight. Supplier clinical safety leads, including those from NEC and InHealth Intelligence, are also being engaged, with initial alignment discussions underway to ensure a shared understanding of the cohorting approach, associated risks, and safety responsibilities.

Reviewer: What is NEC?

## Key improvements enabled by DESP collaboration and pilot approach

Collaboration with DESPs ensures the cohorting model reflects real-world service delivery. The pilot approach enables:

> -	Early identification of risks

Reviewer: ```suggestion
-Early identification of risks
```

- Real-time service feedback
- Iterative refinement of processes and system behaviour
- Increased confidence in clinical safety and operational viability
- Daily updates supported by PDS integration allow faster response to eligibility changes, improving efficiency and patient safety.

## Next steps
-	Finalise pilot onboarding and success criteria 
-	Run controlled pilot and capture feedback
-	Use insights to inform national rollout
-	Prepare for transition ahead of March 2027
-	Continue engagement with DESPs via the CASCADE Operational Working Group

> -	Work with key suppliers NEC and InHealth Intelligence to progress technical integration following Phase 2 planning

Reviewer: ```suggestion
## Next steps
- Finalise pilot onboarding and success criteria 
- Run controlled pilot and capture feedback
- Use insights to inform national rollout
- Prepare for transition ahead of March 2027
- Continue engagement with DESPs via the CASCADE Operational Working Group
- Work with key suppliers NEC and InHealth Intelligence to progress technical integration following phase 2 planning
```
